distribution/lib/Standard/Base/0.0.0-dev/src/Data/Statistics.enso

```diff
@@ -383,6 +389,7 @@ check_if_empty count ~action = if count == 0 then Number.nan else action
 empty_value statistic = case statistic of
     Statistic.Count -> 0
     Statistic.Sum -> 0
+    Statistic.Product -> 0
```

Comment:
Maybe... I couldn't decide what asking for the product of an empty vector actually meant... I can go with it being 1.

Reply:
Yeah, I guess it more or less depends on context.

I was curious what Libre Office would do and indeed it returns `0` for an empty range! So given that our users are familiar with Excel and similar tools it may make sense.

For me, it made sense to go with the 'mathematics'-preferred approach which is `1`.

Given that it is unclear what is the right choice, what if we return `Nothing`? Then the user can consciously fill with whatever makes sense in their context. IMO this has the advantage that given there is no 'default assumption' the user will not have failures stemming from a mismatch between user's assumptions and our assumptions. On the flipside, `Nothing` is probably a little bit more annoying.

Reply:
I like 1 having thought about it some more. But will leave this open until Friday so others can weigh in.

Reply:
Google sheets uses 0 as well, so I think we should go with that. ☹️

Reply:
Yes let's go with the crowd. I put it back to zero.